src/NzbDrone.Common/NzbDrone.Common.csproj

```diff
@@ -244,6 +247,12 @@
   <ItemGroup>
     <Content Include="Expansive\license.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="..\Libraries\Sqlite\libsqlite3.0.dylib">
+      <Link>libsqlite3.0.dylib</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
```

Comment:
is the dylib reference needed since we are just using System.Data.SQLite for the exception type?

Reply:
I'm not totally sure if it's necessary. We have it in Core so thought I should add here if we using sqlite